### ISSUE TITLE
Stringtie reports 0 FPKM reference transcripts with -G mode on 

### ISSUE THREAD

**Issue by qoiopipq** [gpertea/stringtie#102]
Hi,

I used UCSC mm10 reference annotation as a guide to assemble transcripts (with -G mode on) from HISAT2 aligned reads, which I followed the same command (Step 3 assemble and quantify expressed genes and transcripts) you published last year. But I found some assembled transcripts have 0 FPKM values and all of these 0 FPKM transcripts are known/annotated transcripts (687/12,938 known transcripts in the Stringtie's GTF file). One of them as shown below:

chr11	StringTie	**transcript**	70764447	70772928	1000	+	.	gene_id "CD4_DS_uns.10041"; transcript_id "CD4_DS_uns.10041.1"; reference_id "NM_177565"; ref_gene_id "Zfp3"; ref_gene_name "**Zfp3**"; cov "0.000000"; **FPKM "0.000000"**; TPM "0.000000";
chr11	StringTie	**exon**	70764447	70764549	1000	+	.	gene_id "CD4_DS_uns.10041"; transcript_id "CD4_DS_uns.10041.1"; exon_number "1"; reference_id "NM_177565"; ref_gene_id "**Zfp3**"; ref_gene_name "Zfp3"; cov "0.000000";
chr11	StringTie	**exon**	70771209	70772928	1000	+	.	gene_id "CD4_DS_uns.10041"; transcript_id "CD4_DS_uns.10041.1"; exon_number "2"; reference_id "NM_177565"; ref_gene_id "**Zfp3**"; ref_gene_name "Zfp3"; cov "0.000000";

What I read is that using Cufflinks to do reference guided de novo assembly, we would expect to get all reference transcripts no matter what the expression values are plus novel. But in Stringtie, we are expecting to see "expressed" reference transcripts only (plus novel transcripts). So I'm bit confused why I saw some of 0 FPKM reference transcripts in Stringtie as well? I'm using Stringtie v1.2.4. 

Thanks

**Comment by pandafish123** [comment]
have you used a wrong gtf file?

**Comment by qoiopipq** [comment]
I don't think this is the case. I didn't go through all of annotated transcripts with FPKM=0, but I did randomly check around 20 of them and visualised them in IGV. Most of them were supported by aligned reads and spliced junctions (although spliced junctions vary a bit). However, I found one thing very interesting.18 out of 20 annotated transcripts (fpkm=0) are 2-exon transcripts. I don't know if the short annotated transcripts have any impacts on assembling transcripts, which resulting in generating a few false positive transcripts? Or if these short annotated transcripts have any issues with the FPKM calculation step? Or any reasons that I can't think of...
